frontend/syntax.asdl

```diff
@@ -195,11 +195,15 @@ module syntax
      Token right
   )
 
-  AssocPair = (CompoundWord key, CompoundWord value)
+  AssocPair = (CompoundWord key, CompoundWord value, bool has_plus)
```

Comment:
I noticed in `frontend/syntax.asdl` we have `assign_op = Equal | PlusEqual`, and `AssignPair` uses it

In core/runtime.asdl, `AssignArg` has `bool has_plus`

(One of them is static and one is dynamic)

It might be better to be consistent